<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpTopServer</class>
 <widget class="QMainWindow" name="TcpTopServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpTopServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>261</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>打开服务器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前用户</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
